src/components/TableRow/TableRow.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { useState } from 'react'
+import bin from './bin.png'
+
+const TableRow = ({saved, deleteTask}) => {
+
+    const [complete, setComplete] = useState(false)
+
+    const handleChange = () => {
+        setComplete(!complete)
+    }
+
+    
+  return (
+    <tr className={complete ? 'done' : ''} >
+        <td>{saved.category}</td>
+        <td>{saved.task}</td>
+        <td className='center-clm'><button onClick={handleChange} className={complete ? 'check-task checked' : 'check-task' }/></td>
+        <td className='center-clm'><button onClick={() => deleteTask(saved.id)} className='trash bin'><img src={bin} className='bin'/></button></td>
```

Comment:
when you add class of "trash bin", it's 2 classes being applied of "trash" and "bin". If the intention was to have one class, it should be trash-bin?

Reply:
Great call out! I will double check that!